website/docs/reference/terraform.mdx

```diff
@@ -0,0 +1,71 @@
+---
+title: Using Unleash through Terraform
+---
+
+:::note Availability
+
+**Version**: `5.6+`
+
+:::
+
+
+## Overview
+
+The [Unleash Terraform provider](https://github.com/Unleash/terraform-provider-unleash) allows you to manage and configure your Unleash programmatically using Terraform, enabling automated and scalable management for your infrastructure.
+By integrating Unleash with Terraform, you can use infrastructure as code (IaC) to optimize the setup and maintenance of your Unleash instance configuration.
+
+The Unleash Terraform provider is designed to help you with the initial setup and configuration of an instance. The provider does not support managing feature flags through Terraform. Since most [feature flags are short-lived](/topics/feature-flags/feature-flag-best-practices#7-make-flags-short-lived), we recommend managing them through the Unleash Admin UI.
```

Comment:
```suggestion
The Unleash Terraform provider is designed to help you with the **initial setup and configuration** of an instance. The provider does not support managing feature flags through Terraform. Since most [feature flags are short-lived](/topics/feature-flags/feature-flag-best-practices#7-make-flags-short-lived), we recommend managing them through the Unleash Admin UI.
```